bot/exts/info/information.py

```diff
@@ -537,17 +549,44 @@ async def rules(self, ctx: Context, rules: Greedy[int]) -> None:
 
         # Remove duplicates and sort the rule indices
         rules = sorted(set(rules))
```

Comment:
Sorting should happen after all rules, including those from keywords, are found.